ova/provision.sh

```diff
@@ -1,56 +1,129 @@
 #/bin/bash
+
 set -exf
-# Variables
-repo_branch=$(echo "$1" | cut -c1-3)
-repo_baseurl=$(echo "$1" | cut -c1-2)
+
 WAZUH_VERSION=$1
 OPENDISTRO_VERSION=$2
 ELK_VERSION=$3
 PACKAGES_REPOSITORY=$4
 BRANCH=$5
-UI_REVISION=$6
-DIRECTORY="/var/ossec"
-ELK_MAJOR=`echo ${ELK_VERSION}|cut -d"." -f1`
-ELK_MINOR=`echo ${ELK_VERSION}|cut -d"." -f2`
+BRANCHDOC=$6
+INSTALLER="all-in-one-installation.sh"
 
-CURRENT_PATH="$( cd $(dirname $0) ; pwd -P )"
-config_files="${CURRENT_PATH}/Config_files"
-automatic_set_ram_location="/etc/"
-libraries_files="${CURRENT_PATH}/Libraries/"
+# Display dev/prod
+echo "Using ${PACKAGES_REPOSITORY} packages"
 
-echo "${PACKAGES_REPOSITORY}"
+# Create user wazuh - Ignore password error
+adduser wazuh
+yes wazuh | sudo passwd wazuh
 
+# Grant sudo privileges to user
+gpasswd -a wazuh wheel
 
-. ${CURRENT_PATH}/Libraries/provision-opendistro.sh
+# Set Hostname
+hostname wazuhmanager
 
+# Ssh config
+sed -i "s/PasswordAuthentication no/PasswordAuthentication yes/" /etc/ssh/sshd_config
+echo "PermitRootLogin no" >> /etc/ssh/sshd_config
+
+# Download unattended installer
+curl -so ${INSTALLER} https://raw.githubusercontent.com/wazuh/wazuh-documentation/${BRANCHDOC}/resources/open-distro/unattended-installation/${INSTALLER} 
+
+# Get currents version values of installer
```

Comment:
Over-commenting

Reply:
Fixed in https://github.com/wazuh/wazuh-packages/pull/746/commits/6fb50e3584df557334da525324a73a5c32ae0551